osu.Game/Overlays/MusicController.cs

```diff
@@ -441,6 +443,8 @@ private class ProgressBar : SliderBar<double>
 
             private readonly Box fill;
 
+            private bool enabled;

```

Comment:
Shouldn't an enabled property be implemented in the SliderBar class rather than in here?